test/post/controller.js

```diff
@@ -75,19 +45,22 @@ describe('postController', () => {
           sinon.assert.calledWith(postService.getAllPosts);
         })
     ));
-    it('should return resolved result of "postService.getAllPosts" to view', () => (
+    it('should return resolved result of "postService.getAllPosts" as json object', () => (
       request(app)
         .get('/posts')
         .expect(200)
-        .expect(/post1/)
-        .expect(/author1/)
-        .expect(/body1/)
-        .expect(/post2/)
-        .expect(/author2/)
-        .expect(/body2/)
-        .expect(/post3/)
-        .expect(/author3/)
-        .expect(/body3/)
+        .expect('Content-Type', /json/)
+        .expect((res) => {
+          assert.equal(res.body[0].title, 'post1');
+          assert.equal(res.body[0].author, 'author1');
+          assert.equal(res.body[0].body, 'body1');
+          assert.equal(res.body[1].title, 'post2');
+          assert.equal(res.body[1].author, 'author2');
+          assert.equal(res.body[1].body, 'body2');
+          assert.equal(res.body[2].title, 'post3');
+          assert.equal(res.body[2].author, 'author3');
+          assert.equal(res.body[2].body, 'body3');
```

Comment:
Looks like you should add `_id` tests since `_id` is official

Reply:
added _id tests, but I compared res.body._id to mockposts[0]._id. Should I change the title, body, author test to follow the same format?

Reply:
Yes